### PR TITLE
Random nitpicks & doc comments

### PR DESCRIPTION
**Pull request by kohsuke**
... as I was trying to understand this code.

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java [hunk 1]
```diff
@@ -358,19 +364,16 @@ private void doDispatch(@Nonnull Message message) {
...
-        private EventDispatcher eventDispatcher;
-        private int numSubscribers = 0;
 
-        public SSEChannelSubscriber(EventDispatcher eventDispatcher) {
-            this.eventDispatcher = eventDispatcher;
-        }
+    /**
+     * Receive event from {@link PubsubBus} and sends it to this client.
+     */
+    private final class SSEChannelSubscriber implements ChannelSubscriber {
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java]: This is not expected to be `Serializable` I hope?

**Comment by i386** [comment]
Ping @tfennelly

File: src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java (hunk 1 above)
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java]: Not as big problem as https://issues.jenkins-ci.org/browse/JENKINS-41063

Comment (kohsuke) [inline src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java]: I replaced the explicit `eventDispatcher` field by implicit `$0` so no impact on serialization.

**Pull request merged**